src/matchers/toBeArrayOfSize/predicate.js

```diff
@@ -0,0 +1,8 @@
+import toBeArray from '../toBeArray/predicate';
+
+/**
```

Comment:
Is JSDoc OK here?

Reply:
Generally I don't think we have used it elsewhere but I'm OK with it if you want to leave it 👍